pkg/lib/cockpit-po-plugin.js

```diff
@@ -105,7 +103,17 @@ function init(options) {
 }
 
 function run(webpack_module, webpack_compilation) {
-    return Promise.all(get_po_files().map(f => buildFile(f, webpack_module, webpack_compilation)));
+    return Promise.all(get_po_files().map(f => {
+        if (webpack_compilation)
+            webpack_compilation.fileDependencies.add(f);
+        const lang = path.basename(f).slice(0, -3);
+        return Promise.all([
+            buildFile(f, webpack_module, webpack_compilation,
```

Comment:
Maybe a comment here about how these two conditions, although they appear opposite, are not, and how a string might end up in both files (and why we want that).

Reply:
But hmm we do filter out manifest.json for .po.js?